bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -383,16 +385,14 @@ function findOutputApksHelper (dir, build_type, arch) {
 }
 
 function findOutputBundlesHelper (dir, build_type) {
-    var shellSilent = shell.config.silent;
+    const shellSilent = shell.config.silent;
```

Comment:
Nitpicking again:

While I now completely understand from the discussion why we do have the unused `const shellSilent` declaration here, …

… I would now really favor having a comment in the code that explains *why* we have the unused `const` declaration here.

And a warning that yet another nitpick is coming up.

Reply:
I can add a comment along with the other nit.

Reply:
All of the comments here are exactly what I was looking for, yes.